src/pages/hegel/reference/becoming/development.mdx

```diff
@@ -0,0 +1,562 @@
+> same time sublated (Hegel 2010, 80/21.92).
+
+import { Tabs } from "nextra/components";
+
+<Tabs items={['Main reading', 'Alternative reading']}>
+
+<Tabs.Tab data-name="main reading">
+
+Here Hegel stresses that `becoming` is _not_ to be understood as just the a
+static unity that merely combines `being` and `nothing` side-by-side without
+taking into account the nuances of their respective developments. Inasmuch as
+`becoming` is regarded as the unity of the two categories it has specifically to
+do with their inseparability.
+
+However, this inseparability needs to be further qualified. If each category is
+isolated with this inseparability, then it is negated on account that it does
+not have any self-subsistence, reality or independence vis-à-vis the other it is
+inseparable from.
+
+Therefore, _the unity of_ `becoming` contains both categories as _moments_. This
+sublated state does double work: first, it ensures the distinctiveness of each
+category; second, it _also_ illustrates their inseparability through the shared
+context.
+
+Put differently, the developments of `being` and `nothing` lead to divergent
+results: on the one hand, each is an immediate self-subsistent category totally
+distinct from the other, but, on the other hand, each is indistinguishable from
+the other and vanishes into that other, implying an undeniable inseparability.
+This looks like a problem: How can these results be coherently understood
+together? `Becoming` is the category that creatively combines these two results
+through the idea of "moment": a logic that at once joins together while it
+differentiates.
+
+</Tabs.Tab>
+
+<Tabs.Tab data-name="alternate reading">
+
+Here `becoming` is further developed into two phases. First, as the
+"unseparatedness" of `being` and `nothing`. Secondly, as the "unity" of `being`
+and `nothing`. Hegel makes this distinction within `becoming` following the
+inseparability and difference exposed in the
+[first section](#the-unity-of-being-and-nothing) since each of these signal a
+different conception with regards to how the category is understood.
+
+For example, insofar as `being` is understood to be inseparable from `nothing`,
+`being` does not have any self-subsistence, reality or independence and so its
+meaning of `pure being` is negated and with it the category itself ("each is
+not"). And, conversely, insofar as `being` is understood to be distinct from
+`nothing`, it is joined into a determinate unity with `nothing` where the two
+"equally are". In the second phase, Hegel adds additional qualifiers: each
+category is present equally as the other yet only as vanishing and sublated.
+This is important because the categories of `being` and `nothing` are not held
+together immediately&mdash;`pure being` and `pure nothing` strictly cannot
+co-exist&mdash;but as mediated moments. In other words, they are not held
+together in their initial immediacy but as they disappear and are turned into
+moments.
```

Comment:
very well explicated. The distinction between them being united as co-existing and being united as vanishing moments is well explained.